packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -271,6 +291,24 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
     );
   }
 
+  private _renderSeparator(item: ICommandBarItemProps, index: number, isFarItem?: boolean): React.ReactNode {
+    // Do not render if separator is first or last item
+    if (index > 0 &&
```

Comment:
I'm not sure we need this conditional. If the user doesn't want to add a separator at the start or end, I feel like it;s up to them to verify that.

Reply:
Context menu checks for dividers at the start of their items and doesn't render them. I'm assuming that there is a reason why contextmenu does that. Otherwise, I can remove in both locations.

Reply:
The check for if it is the last item is so that we don't render a divider between the last item and the overflow expander.